app/src/main/java/org/mifos/mobile/viewModels/RegistrationViewModel.kt

```diff
@@ -0,0 +1,65 @@
+package org.mifos.mobile.viewModels
+
+import android.util.Patterns
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.observers.DisposableObserver
+import io.reactivex.schedulers.Schedulers
+import okhttp3.ResponseBody
+import org.mifos.mobile.api.DataManager
+import org.mifos.mobile.models.register.RegisterPayload
+import javax.inject.Inject
+
+class RegistrationViewModel @Inject constructor(var dataManager : DataManager?) : ViewModel() {
```

Comment:
Repository??? Please create a Repository interface and its implementation.

Reply:
Jawid I created the RegistrationRepository Class without the RegistrationRepository Interface. Reason being we only have one source(i.e. remote source) for data fetching, so I found it redundant to create only one interface  file RemoteRegistrationRepositoryInterface and then implement it, as we don't have local or test source. If you still insist that we go with an interface, I'll do that. Thanks.